plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -292,14 +292,43 @@ struct Database {
 
     /// Add a mutation score to the history table.
     void putMutationScore(const MutationScore score) @trusted {
-        db.run(insert!MutationScoreHistoryTable, MutationScoreHistoryTable(0,
-                score.timeStamp, score.score.get));
+        auto stmt = db.prepare(format!"SELECT id FROM " ~ mutationScoreHistoryTable ~ "
```

Comment:
i think this select is a left-over from implementing the new putMutationScore. it isn't needed anymore?